Marlin/Configuration.h

```diff
@@ -97,7 +110,7 @@ Here are some standard links for getting your machine calibrated:
 
 // Optional custom name for your RepStrap or other custom machine
 // Displayed in the LCD "Ready" message
-//#define CUSTOM_MACHINE_NAME "3D Printer"
+#define CUSTOM_MACHINE_NAME "Prusa i3"
```

Comment:
@thinkyhead maybe you committed your own config file ?
This seems to be changing the default config file a lot.